app/config/Config.scala

```diff
@@ -18,24 +17,16 @@ object Config extends AwsInstanceTags {
     new InstanceProfileCredentialsProvider(false)
   )
 
-  val configMagicMode = stage match {
-    case "DEV" => Mode.Dev
-    case "CODE" => Mode.Prod
-    case "PROD" => Mode.Prod
-    case _ => sys.error("invalid stage")
-  }
-  val config = ConfigurationMagic(appName, configMagicMode).load
-
-  def getOptionalProperty[T](path: String, getVal: String => T) = {
-    if (config.hasPath(path)) Some(getVal(path))
-    else None
-  }
-
-  val elkKinesisStream = config.getString("elk.kinesis.stream")
-  val elkLoggingEnabled = getOptionalProperty("elk.logging.enabled", config.getBoolean).getOrElse(true)
-
-  val pandaDomain = config.getString("panda.domain")
-  val pandaAuthCallback = config.getString("panda.authCallback")
-  val pandaSystem = config.getString("panda.system")
+  def getConfigString(key: String): String = config.getString(key)
+    .getOrElse(throw new RuntimeException(s"key $key not found in configuration"))
+
+  val elkKinesisStream = getConfigString("elk.kinesis.stream")
+  val elkLoggingEnabled = config.getBoolean("elk.logging.enabled").getOrElse(true)
+
+  val pandaDomain = getConfigString("panda.domain")
+  val pandaAuthCallback = getConfigString("panda.authCallback")
+  val pandaSystem = getConfigString("panda.system")
+
+  val dbConfig = config.underlying.getConfig("ctx")
```

Comment:
`underlying` needs to be used here as the dbConfig is passed into the `PostgresJdbcContext` object in the `MetricsDB`. `PostgresJdbcContext` requires something of type `com.typesafe.config.Config`  not a `play.api.Configuration`